### ISSUE TITLE
Horizontal rule runs over thumbnail on mobile

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/SearchWorks#4220]
![image](https://github.com/sul-dlss/SearchWorks/assets/92044/cd927d0b-828c-49cd-8849-d32d4df44aed)

**Comment by dnoneill** [comment]
This is true at certain sizes on regular size screens as well. It is really hard to get right because of the col breakpoints.

**Comment by saseestone** [comment]
Closing because we're going to work on 4.0.
